.github/workflows/verify_tutorial_toc.py

```diff
@@ -0,0 +1,46 @@
+import yaml
+from pathlib import Path
+
+
+def get_yaml_links():
```

Comment:
Maybe it would be cleaner to have that as a pytest test in `tools/testing/check_source_code.py`? Or in a separate file in this directory? Then we run everything with pytest in one single Github actions job.

Reply:
Good idea,  will refactor the test to be in standard testing location.